### PR TITLE
[getting-started] update extension readme following updated template

### PR DESCRIPTION
**Pull request by vince-fugnitto**
#### What it does

Updates the `getting-started` extension readme following the new template.
The new template consists of:
- extension header
- extension description
- extension docs: generated API documentation
- extension license
- extension trademark

#### How to test

Review [rendered](https://github.com/eclipse-theia/theia/blob/d66365f86cfd6b51a687c962fb1180f1fadad275/packages/getting-started/README.md) readme content.

#### Review checklist

- [x] as an author, I have thoroughly tested my changes and carefully followed [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#requesting-a-review)

#### Reminder for reviewers

- as a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)

### CODE REVIEW

**Comment by vince-fugnitto** [comment]
This is a tentative template regarding improvements for https://github.com/eclipse-theia/theia/issues/6518. The goal is to provide a readme which contains the necessary information to understand an extension at a high-level and not require constant updates.

The template includes:
- an update header (mainly for aesthetic purposes as the `readme` is displayed on `npm`)
- a high-level description of the extension
- a link to the generated documentation for the readme
- license section
- trademark section

I welcome any feedback :smiley:

**Comment by marcdumais-work** [comment]
Very good start - I like the (limited) level of details - there should be little need to update unless there are major changes in the extension. The presentation is nice as well. 

@eclipse-theia/core : Please have a look and comment. Once we start updating the various extension's `README.md` in the same way, it will be too late to suggest changes to the template :)

File: logo/theia.svg [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000" height="1000">
+<!--
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
```
[the remark this answers is not in the document]
Comment (vince-fugnitto) [reply logo/theia.svg]: I kept the original header, the icon is copied from `theia-cpp-extensions` for example.
I'm not sure what the process should be for such a thing.

File: packages/getting-started/README.md [hunk 2]
```diff
@@ -1,15 +1,40 @@
...
 - `open commands`: commands which are used to open files, folders, and workspaces quickly.
 - `recent workspaces`: recently used workspaces are listed for easy and quick access.
 - `settings commands`: commands which are used to open the preferences and keyboard shortcuts widgets.
 - `help`: useful links pointing to documentation and/or guides.
 
+
+## Additional Information
+
+- [API documentation for `@theia/getting-started`](https://eclipse-theia.github.io/theia/docs/next/modules/getting_started.html)
+- [Theia - GitHub](https://github.com/eclipse-theia/theia)
```
Comment (vince-fugnitto) [inline packages/getting-started/README.md]: @lmcbout I added the following links to the `Additional Information` section for the `readme`.
Did you have something else in mind?

File: logo/theia.svg (hunk 1 above)
[the remark this answers is not in the document]
Comment (marcdumais-work) [reply logo/theia.svg]: > I kept the original header, the icon is copied from `theia-cpp-extensions` for example.
> I'm not sure what the process should be for such a thing.

We are allowed to copy content between Eclipse Foundation projects, even more so within the same project. I do not think there is anything else to do. I think keeping the original header in that case is the way to go.

File: packages/getting-started/README.md [hunk 3]
```diff
@@ -1,15 +1,40 @@
...
+- [Theia - Website](https://theia-ide.org/)
+
 ## License
-- [Eclipse Public License 2.0](http://www.eclipse.org/legal/epl-2.0/)
-- [一 (Secondary) GNU General Public License, version 2 with the GNU Classpath Exception](https://projects.eclipse.org/license/secondary-gpl-2.0-cp)
+
+- [Eclipse Public License 2.0](LICENSE)
+- [一 (Secondary) GNU General Public License, version 2 with the GNU Classpath Exception](LICENSE)
```
Comment (lmcbout) [inline packages/getting-started/README.md]: Why did you remove then link for the secondary liscense (https://projects.eclipse.org/license/secondary-gpl-2.0-cp)

Comment (vince-fugnitto) [reply packages/getting-started/README.md]: Thanks for spotting the error, it was a copy+paste problem :)

**Comment by vince-fugnitto** [comment]
I haven't gotten much feedback regarding the new extension template so I'll assume that the content is fine, I'll proceed with updating the existing extension's `readme` (and will exclude soon to be deprecated extension as described in #6933).
